packages/code-du-travail-api/src/server/routes/__tests__/suggest.spec.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const Koa = require("koa");
+const router = require("../suggest");
+
+const app = new Koa();
+app.use(router.routes());
+
+function getSuggestions(query) {
+  return request(app.callback()).get(`/api/v1/suggest?q=` + query);
+}
+
+test("return suggestions for re in the right format", () =>
+  getSuggestions("re")
+    .expect(200)
+    .expect("Content-Type", /json/)
+    .expect(res => expect(res.body).toMatchSnapshot()));
+
+test("accentuation is ignored", async () =>
+  getSuggestions("ré").expect(res =>
+    // this is ugly, coulnt find a better way though
+    expect(res.body).toEqual(expect.arrayContaining(["retraite"]))
```

Comment:
I would write it in natural javascript `expect(res.body.includes("retraite")).toBeTruthy()`. Does that looks better to you ?

Reply:
Or maybe this will work too: `expect(res.body).toContain("retraite")`

Reply:
you could first use await (you already added the `async`)
```js
test("accentuation is ignored", async () => { 
  const response = await getSuggestions("ré")
  expect(response.body.include("retraite")).toBeTruthy()
  // see https://jestjs.io/docs/en/using-matchers.html#arrays-and-iterables
  expect(res.body).toContain("retraite"
}
```